src/message/MessageFull.js

```diff
@@ -74,6 +74,7 @@ export default class MessageFull extends PureComponent<Props> {
           email={message.sender_email}
           name={message.sender_full_name}
           onPress={this.handleAvatarPress}
+          size={40}
```

Comment:
Checked on other apps.
Slack too have avatar of size 40.
And our current is somewhat small on physical size.